src/applications/search/tests/e2e/00-required.cypress.spec.js

```diff
@@ -0,0 +1,90 @@
+import stub from '../../constants/stub.json';
+
+const SELECTORS = {
+  APP: '[data-e2e-id="search-app"]',
+  SEARCH_FORM: '[data-e2e-id="search-form"]',
+  SEARCH_RESULTS: '[data-e2e-id="search-results"]',
+  SEARCH_RESULTS_EMPTY: '[data-e2e-id="search-results-empty"]',
+  SEARCH_RESULT_TITLE: '[data-e2e-id="result-title"]',
```

Comment:
Did you mean to use this selector [here](https://github.com/department-of-veterans-affairs/vets-website/pull/16476/files#diff-635826737dc9cad89239ddd597ecfdfe598c495095120dfc1c1af0d41b3985bdR49)? If not, you could probably remove it 🙂